### ISSUE TITLE
No main region landmark is implemented on the page

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1360]
**Actual Result:**
No main region landmark is implemented on the page.

**Expected Result:**
Main region landmark should be implemented on the page.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Start Narrator by pressing "Ctrl + Win + Enter" key.
3. Navigate to 'Type your message' text box present in bottom of screen, type 'help' and press enter key.
4. Navigate to any button and activate it.
5. Press 'D' key to navigate the page in landmark mode.
6. Observe if main region landmark is implemented on page or not.

**User Impact:**
If main region landmark is not implemented on page then the user who dependent on screen reader will not able to expose visual groupings and semantic structure of controls and content programmatically.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge

**MAS Reference:**
[MAS 2.4.1 - Bypass Blocks (27)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B20bce2aa-9952-4a10-af79-3376bcc8766d%7D)

**Attachment:**
[MAS2.4.1_Landmark.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2584601/MAS2.4.1_Landmark.zip)

**Comment by compulim** [comment]
- [ ] Put `role=complementary` on the top container of Web Chat
- [ ] Put `role=form` on the send box of Web Chat
- [ ] Put `role=region` on the transcript of Web Chat
- [ ] On all the samples, add `role=main` to first child of `<body>`

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG2.4.1
